rsts/concepts/schedules.rst

```diff
@@ -0,0 +1,20 @@
+.. _concepts-schedules:
+
+Schedules
+---------
+Workflows can be run automatically using schedules associated with launch plans. Schedules can either define a cron_expression_. or rate_unit_.
+
+At most one launch plan version for a given {Project, Domain, Name} combination can be active, which means at most one schedule can be active for a launch plan. However, many unique launch plans and corresponding schedules can be defined for the same workflow.
+
+.. _cron_expression:
+
+Cron Expression
+---------------
+Cron expression strings use the `AWS syntax <http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#CronExpressions>`__. These are validated at launch plan registration time.
```

Comment:
This is actually wrong and now we prefer using just regular cron syntax. 
Here is some doc. https://github.com/flyteorg/flyteadmin/blob/master/scheduler/doc.go#L53-L55

Can you please update it? Also talk to @pmahindrakar-oss for more information. I think we should also add examples of each

Reply:
Sure Ketan

Reply:
We can add this table
CRON Expression Format ¶
A cron expression represents a set of times, using 5 space-separated fields.

```
Field name   | Mandatory? | Allowed values  | Allowed special characters
----------   | ---------- | --------------  | --------------------------
Minutes      | Yes        | 0-59            | * / , -
Hours        | Yes        | 0-23            | * / , -
Day of month | Yes        | 1-31            | * / , - ?
Month        | Yes        | 1-12 or JAN-DEC | * / , -
Day of week  | Yes        | 0-6 or SUN-SAT  | * / , - ?
```

Then short hand cron formats / predefined schedules
```
Entry                  | Description                                | Equivalent To
-----                  | -----------                                | -------------
@yearly (or @annually) | Run once a year, midnight, Jan. 1st        |  0 0 1 1 *
@monthly               | Run once a month, midnight, first of month |  0 0 1 * *
@weekly                | Run once a week, midnight between Sat/Sun  |  0 0 * * 0
@daily (or @midnight)  | Run once a day, midnight                   |  0 0 * * *
@hourly                | Run once an hour, beginning of hour        |  0 * * * *

```

Which needs be populated in the string field of the schedule eg : here https://docs.flyte.org/projects/cookbook/en/latest/auto/core/scheduled_workflows/lp_schedules.html#cron-schedules

eg :
```
cron_lp_every_min_of_hour = LaunchPlan.get_or_create(
    name="my_cron_scheduled_lp",
    workflow=date_formatter_wf,
    schedule=CronSchedule(
        # Note that kickoff_time_input_arg matches the workflow input we defined above: kickoff_time
        # But in case you are using the AWS scheme of schedules and not using the native scheduler then switch over the schedule parameter with cron_expression
        schedule="*/1 * * * *", # Following schedule runs every min
        kickoff_time_input_arg="kickoff_time",
    ),
)

cron_lp_every_min_of_hour = LaunchPlan.get_or_create(
    name="my_cron_scheduled_lp",
    workflow=date_formatter_wf,
    schedule=CronSchedule(
        # Note that kickoff_time_input_arg matches the workflow input we defined above: kickoff_time
        # But in case you are using the AWS scheme of schedules and not using the native scheduler then switch over the schedule parameter with cron_expression
        schedule="@hourly", # Following schedule runs every hour at beginning of the hour
        kickoff_time_input_arg="kickoff_time",
    ),
)
```

For fixed rate schedules
https://docs.flyte.org/projects/cookbook/en/latest/auto/core/scheduled_workflows/lp_schedules.html#fixed-rate-intervals
We support
```
duration=timedelta(minutes=10)
timedelta with minutes,hours,days and weeks
```

So we can add examples for fixed rate with

```
fixed_rate_lp_minutes = LaunchPlan.get_or_create(
    name="my_fixed_rate_lp_minutes",
    workflow=positive_wf,
    # Note that the workflow above doesn't accept any kickoff time arguments.
    # We just omit the ``kickoff_time_input_arg`` from the FixedRate schedule invocation
    schedule=FixedRate(duration=timedelta(minutes=10)),
    fixed_inputs={"name": "you"},
)

fixed_rate_lp_days = LaunchPlan.get_or_create(
    name="my_fixed_rate_lp_days",
    workflow=positive_wf,
    # Note that the workflow above doesn't accept any kickoff time arguments.
    # We just omit the ``kickoff_time_input_arg`` from the FixedRate schedule invocation
    schedule=FixedRate(duration=timedelta(days=1)),
    fixed_inputs={"name": "you"},
)

fixed_rate_lp_hours = LaunchPlan.get_or_create(
    name="my_fixed_rate_lp_hours",
```